model-optimizer/extensions/front/tf/reduce_ext.py

```diff
@@ -67,3 +67,13 @@ class SumFrontExtractor(FrontExtractorOp):
     def extract(cls, node: Node):
         ReduceSum.update_node_stat(node, {'keep_dims': node.pb.attr["keep_dims"].b})
         return cls.enabled
+
+
+class EuclideanNormFrontExtractor(FrontExtractorOp):
+    op = 'EuclideanNorm'
```

Comment:
The TF EuclideanNorm operation has `axis` as input (not attribute). So we need another operation where `axis` is input.

Reply:
But in TF [specification](https://www.tensorflow.org/api_docs/python/tf/math/reduce_euclidean_norm) tf.math.reduce_euclidean_norm operation has `axis` as attribute, not as input

Reply:
From the specification that you mention it is not clear whether it is input or attribute. But if you take a look at the TF [source code](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/math_ops.cc#L999) you will find the operation declaration:
```
REGISTER_OP("EuclideanNorm")
    .Input("input: T")
    .Input("reduction_indices: Tidx")
    .Output("output: T")
    .Attr("keep_dims: bool = false")
    .Attr("T: numbertype")
    .Attr("Tidx: {int32, int64} = DT_INT32")
    .SetShapeFn(shape_inference::ReductionShape);
```
and here you can see that `reduction_indices` is an input.